jobs/ci-run/build/builddqlite.yml

```diff
@@ -18,8 +18,6 @@
           regex: ^\S{7}$
           msg: Enter a valid 7 char git sha
     builders:
-      - get-build-details
-      - set-test-description
```

Comment:
We don't need juju to be built before building dqlite, so remove this dependency.